flink-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/proxy/KinesisProxy.java

```diff
@@ -262,8 +262,8 @@ public GetRecordsResult getRecords(String shardIterator, int maxRecordsToGet) th
 		}
 
 		if (getRecordsResult == null) {
-			throw new RuntimeException("Rate Exceeded for getRecords operation - all " + getRecordsMaxRetries +
-				" retry attempts returned ProvisionedThroughputExceededException.");
+			throw new RuntimeException("Retries exceeded for getRecords operation - all " + getRecordsMaxRetries +
```

Comment:
Please also fix same issue further down in `getShardIterator` (https://github.com/apache/flink/pull/7706/files#diff-ed02b5340df65de06c19eb93fe90a920L340)

Reply:
Done.